rflx/model/type_.py

```diff
@@ -511,6 +555,10 @@ def type_(self) -> rty.Type:
     def element_size(self) -> expr.Expr:
         return expr.Number(8)
 
+    @property
+    def serialize(self) -> Dict[str, Any]:
+        return {"kind": self.__class__.__name__, "data": {}}
+
```

Comment:
What is the `data` key for?

Reply:
Three elements (`Opaque`, `BooleanTrue` and `BooleanFalse`) have an empty `data` element (for the boolean elements this is due to the common `serialize` filtering out all dict items). In the Ada import I check the outer structure (`kind` and `data`) generically and then dispatch on the value of `kind` to parse `data`. Having such a generic structure keeps (the error handling of) the parser simple. Removing empty `data` sections would required additional special cases.